### PR TITLE
Develop

### PR DESCRIPTION
**Pull request by VitaliyFrolov**
added route to pages

### CODE REVIEW

File: src/app/App.tsx [hunk 1]
```diff
@@ -32,8 +35,16 @@ const router = createBrowserRouter([
...
-        element: <ShopsPage />
+        path: '/stores',
+        element: <StoresPage />,
+      },
+      {
+        path: '/stores/:store',
+        element: <StorePage />
+      },
+      {
+        path: '/stores/:store/products/:product',
```
Comment (creator-woow) [inline src/app/App.tsx:46]: storeId productId

File: src/app/lib/routes.ts [hunk 2]
```diff
@@ -1,7 +1,8 @@
 export enum PagePath {
   Home = '/',
-  Stores = '/shops',
-  Store = '/stores/:id',
+  Stores = '/stores',
+  Store = '/stores/:store',
+  Product = '/stores/:store/products/:product',
```
Comment (creator-woow) [inline src/app/lib/routes.ts:5]: same

File: src/pages/Home/ui/StoreCard.module.scss [hunk 3]
```diff
@@ -7,4 +7,9 @@
     font-size: 12px;
     font-weight: 400;
     color: rgba(149, 149, 149, 1);
-}
\ No newline at end of file
+}
+
+.link {
+    text-decoration: none;
```
Comment (creator-woow) [inline src/pages/Home/ui/StoreCard.module.scss:13]: to reset

File: src/pages/Home/ui/StoreCard.tsx [hunk 4]
```diff
@@ -1,23 +1,27 @@
...
 
 export interface IStoreCardProps extends ICardProps {
     discountsCount: number;
     name: string;
+    link?: string;
 }
 
 export const StoreCard: FC<IStoreCardProps> = (props) => {
-    const { discountsCount, name, imgUrl, className } = props
+    const { discountsCount, name, imgUrl, link, className } = props
```
Comment (creator-woow) [inline src/pages/Home/ui/StoreCard.tsx:13]: url or path or something like that

File: src/pages/Home/ui/StoreCard.tsx [hunk 5]
```diff
@@ -1,23 +1,27 @@
...
     return (
-        <Card imgUrl={imgUrl} imgAlt={name} className={className}>
-            <h3 className={styles.discountsCount}>
-                {discountsCount} Акций
-            </h3>
-            <p className={styles.name}>
-                {name}
-            </p>
-        </Card>
+        <Link to={`${link}`} className={styles.link}>
```
Comment (creator-woow) [inline src/pages/Home/ui/StoreCard.tsx:16]: delete ``

File: src/pages/Home/ui/StoresList.tsx [hunk 6]
```diff
@@ -19,6 +19,7 @@ export const StoresList: FC<IStoresListProps> = (props) => {
             name={item.name}
             discountsCount={item.discountsCount}
             imgUrl={item.imgUrl}
+            link={item.link}
```
Comment (creator-woow) [inline src/pages/Home/ui/StoresList.tsx:22]: up

File: src/pages/Product/ui/Page.tsx [hunk 7]
```diff
@@ -0,0 +1,16 @@
+import { FC } from 'react';
+import { useParams } from 'react-router';
+import { Container } from 'shared/ui/Container';
+import styles from './Page.module.scss';
+
+export const Page: FC = () => {
+    const { store, product } = useParams();
```
Comment (creator-woow) [inline src/pages/Product/ui/Page.tsx:7]: up

File: src/pages/Store/ui/Page.module.scss [hunk 8]
```diff
@@ -0,0 +1,60 @@
...
+}
+
+.sectionFooter {
+    margin-top: 40px;
+    display: flex;
+    align-items: center;
+    justify-content: center;
+}
+
+.filterWrapper {
```
Comment (creator-woow) [inline src/pages/Store/ui/Page.module.scss:18]: filter ubrat

File: src/pages/Store/ui/Page.module.scss [hunk 9]
```diff
@@ -0,0 +1,60 @@
...
+    margin-bottom: 56px;
+    box-shadow: 0px 8px 20px 0px rgba(0, 0, 0, 0.15);
+    
+    display: flex;
+    flex-wrap: wrap;
+    position: relative;
+    justify-content: space-between;
+}
+
+.searchForm {
```
Comment (creator-woow) [inline src/pages/Store/ui/Page.module.scss:31]: search already exists check ui/shared

File: src/pages/Store/ui/Page.module.scss [hunk 10]
```diff
@@ -0,0 +1,60 @@
...
+    padding-left: 14.5px;
+    border: none;
+    outline: none;
+}
+
+.inputBtn {
+    height: 99%;
+    padding: 10px 29px 10px 28px;    
+}
+
```
Comment (creator-woow) [inline src/pages/Store/ui/Page.module.scss:56]: same

File: src/pages/Store/ui/Page.tsx [hunk 11]
```diff
@@ -0,0 +1,78 @@
...
+import { Search } from "shared/ui/Search";
+import { dataURL } from "pages/Stores/lib/dataURL";
+
+export const Page: FC = () => {
+    const {store} = useParams();
+    const [discounts, setDiscounts] = useState<IDiscountCardProps[]>([]);
+    const [fetching, setFetching] = useState(true);
+    const [filter, setFilter] = useState(`http://localhost:3005/${store}`)
+    
+    useEffect(() => {
```
Comment (creator-woow) [inline src/pages/Store/ui/Page.tsx:21]: infinity scroll delete

File: src/pages/Store/ui/Page.tsx [hunk 12]
```diff
@@ -0,0 +1,78 @@
...
+                        <Search className={styles.input} placeholder='Поиск магазинов' />
+                        <Button className={styles.inputBtn}>
+                            Найти
+                        </Button>
+                    </form>
+                </div>
+                <Checkbox
+                    text="Товары только со скидкой"
+                />
+                <div className={styles.filterWrapper}>
```
Comment (creator-woow) [inline src/pages/Store/ui/Page.tsx:59]: already create filter in features/stores

File: src/pages/Stores/lib/dataURL.ts [hunk 13]
```diff
@@ -1,4 +1,5 @@
 export enum dataURL {
```
Comment (creator-woow) [inline src/pages/Stores/lib/dataURL.ts:1]: rename

File: src/pages/Stores/ui/Page.tsx [hunk 14]
```diff
@@ -35,7 +34,6 @@ export const Page: FC = () => {
                     <Title>
                         Магазины
                     </Title>
-                    <Checkbox text="Товары только со скидкой"/>
```
Comment (creator-woow) [inline src/pages/Stores/ui/Page.tsx:38]: rename to label or caption

File: src/shared/ui/Title/ui/Title.tsx [hunk 15]
```diff
@@ -1,10 +1,10 @@
-import { FC } from "react";
+import { FC, ReactElement } from "react";
 import cn from 'classnames';
 import styles from './Title.module.scss';
 
 export interface ITitleProps {
     className?: string;
-    children: string;
+    children: string | ReactElement;
```
Comment (creator-woow) [inline src/shared/ui/Title/ui/Title.tsx:7]: string only

**Review by creator-woow — APPROVED** [review]
fix that shish

**Review by creator-woow — CHANGES_REQUESTED** [review]
fix that shish

File: src/app/App.tsx [hunk 16]
```diff
@@ -32,8 +35,16 @@ const router = createBrowserRouter([
         element: <HomePage />
       },
       {
-        path: '/shops',
-        element: <ShopsPage />
+        path: '/stores',
```
Comment (creator-woow) [inline src/app/App.tsx:38]: use enum with paths

File: src/features/StoresList/ui/StoreCard.tsx [hunk 17]
```diff
@@ -1,23 +1,27 @@
...
     return (
-        <Card imgUrl={imgUrl} imgAlt={name} className={className}>
-            <h3 className={styles.discountsCount}>
-                {discountsCount} Акций
-            </h3>
-            <p className={styles.name}>
-                {name}
-            </p>
-        </Card>
+        <Link to={`${url}`} className={styles.link}>
```
Comment (creator-woow) [inline src/features/StoresList/ui/StoreCard.tsx:16]: delete ``

File: src/pages/Product/ui/Page.tsx (hunk 7 above)
Comment (creator-woow) [inline src/pages/Product/ui/Page.tsx:7]: rename params names

File: src/pages/Store/ui/Page.module.scss [hunk 18]
```diff
@@ -0,0 +1,65 @@
...
+    margin-top: 60px;
+    margin-bottom: 153px;
+}
+
+.sectionHeader {
+    margin-top: 41px;
+    margin-bottom: 40px;
+}
+
+.sectionFooter {
```
Comment (creator-woow) [inline src/pages/Store/ui/Page.module.scss:11]: delete filter and search input styles, it's styles in components

File: src/pages/Store/ui/Page.tsx [hunk 19]
```diff
@@ -0,0 +1,62 @@
...
+import { getRegistryData } from "pages/Stores/lib/dataGetters";
+
+export const Page: FC = () => {
+    const {storeId} = useParams();
+    const [discounts, setDiscounts] = useState<IDiscountCardProps[]>([]);
+    const [fetching, setFetching] = useState(true);
+    const [filter, setFilter] = useState(`http://localhost:3005/${storeId}`)
+    const [categories, setCategories] = useState([]);
+
+    useEffect(() => {
```
Comment (creator-woow) [inline src/pages/Store/ui/Page.tsx:21]: delete infinity scroll preparation

**Review by creator-woow — CHANGES_REQUESTED** [review]

**Comment by VitaliyFrolov** [comment]
added path to page

**Pull request merged**